### ISSUE TITLE
Advanced Alchemy Furnace stops outputting. Output "contains 0 machina essentia"

### ISSUE THREAD

**Issue by justicesays** [Azanor/thaumcraft#1596]
My advanced alchemical furnace stopped outputting, it has access to a full supply of CV (13/11/11).
All my suction on the outputs seems fine.
When I hit the output pipe section of the AAF it claims "Contains 0 Machina essentia" , 0 suction.
Looks like a 0 value bug somewhere.
I guess I'm going to have to destroy it and re-create it as cycling the server didnt help.

**Comment by Azanor** [comment]
You probably are. Bugs like that aren't supposed to happen, but now and again they do creep in.
I'll take a look and see if I can track down the cause, though these kinds of bugs tend to be hard to track down :(

**Comment by mdraves91** [comment]
I ran into this bug with 0 ignis essentia instead. I was able to reproduce it consistently with the following steps:

1) Construct the AAF
2) Throw in a vanilla minecraft furnace
3) 6 terra and 6 perdito will be output
4) 0 ignis will now be stuck in the output pipe section.

**Comment by Islid** [comment]
I likewise have run into this issue, and am able to reproduce it in the same way that mdraves91 is. It also seems to happen with a bunch of other random items that have perdito (I can't be absolutely sure that's the common thread, but based on some testing that seems to be it). It's happened four times to me now in the last week.

**Comment by seebs** [comment]
Yeah, I just hit this, and I am pretty sure I also hit it because of an iron-capped wooden wand. Ended up with a spare, dumped it in with a bunch of items.

I do not know of a way to unstick the furnace at this point.

**Comment by seebs** [comment]
And of course, I found a way. Checked the contents, it said "0 Cognito".

Dumped in a piece of paper. Now it's working.
